test/python/WMCore_t/MicroService_t/MSUnmerged_t/MSUnmergedRSE_t.py

```diff
@@ -0,0 +1,69 @@
+"""
+Unit tests for MicorService/MSUnmerged/MSUnmergedRSE.py module
+
+"""
+from __future__ import division, print_function
+
+import json
+import os
+import unittest
+import mongomock
+
+from pymongo import IndexModel
+
+from WMCore.MicroService.MSUnmerged.MSUnmerged import MSUnmerged, MSUnmergedRSE
+
+
+class MSUnmergedRSETest(unittest.TestCase):
+    """ Unit test for MSUnmergedRSE module """
+    # pylint: disable=W0212,C0301
```

Comment:
Todor, can you please remove these pylint messages disabled here? If it's not simple to fix them, it's better to let them to get publish such that one day we can come back to them.